### DISCUSSION TITLE
Unable to create webhook

### DISCUSSION THREAD

**Discussion by HFulcher** [def-studio/telegraph#385] [Q&A] [answered]
Creating a webhook via artisan locally is giving me the following error:

`You application must have a secure (https) url in order to accept webhook calls`

I understand that https is needed but the locally run site (via valet) has https with a self-signed cert. What do I need to do to get this to work?

**Comment by fabio-ivona** [comment]
hi @HFulcher !

you should ensure that your application returns an url like this: `https://......` when you call this route:

`route('telegraph.webhook', $botId)`

also, your application must be visible from outside your local network, so Telegram can reach you with webhooks

**Reply by HFulcher** [reply]
Hey @fabio-ivona thanks for the response. I think the issue will be that the app isn't visible from outside the local network as it's just a local development version. So I can just deploy to prod and run the commands there to get things setup?

**Reply by fabio-ivona** [reply, marked as answer]
yep, just check your APP_URL env variable to begin with `https`

**Reply by HFulcher** [reply]
Gotcha, thanks
